audio/nrsc5/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        theori-io nrsc5 f1a32a551471dc5045193027e4c828746c5790fd
```

Comment:
A monotonically increasing `version` needs to be set. If upstream doesn't have a version scheme in place, something like `0.0-20230411` will do, with the date stamp changing to the date of future commits that are used.

Reply:
I have added a version based on your suggestion